accounting/service.proto

```diff
@@ -9,25 +9,24 @@ import "accounting/types.proto";
 import "refs/types.proto";
 import "session/types.proto";
 
-// The service provides methods for obtaining information
-// about the account balance in NeoFS system.
+// Accounting service provides methods for interaction with NeoFS sidechain via
+// other NeoFS nodes to get information about the account balance. Deposit and
+// Withdraw operations can't be implemented here, as they require Mainnet NeoFS
+// Smart Contract invocation.
 service AccountingService {
-  // Returns the amount of funds for the requested NeoFS account.
+  // Returns the amount of funds in GAS token for the requested NeoFS account.
   rpc Balance (BalanceRequest) returns (BalanceResponse);
 }
 
-// Message defines the request body of Balance method.
-//
-// To indicate the account for which the balance is requested, it's identifier
-// is used.
-//
-// To gain access to the requested information, the request body must be formed
-// according to the requirements from the system specification.
+// BalanceRequest message
 message BalanceRequest {
-  //Request body
+  // To indicate the account for which the balance is requested, it's identifier
+  // is used. It can be any existing account in NeoFS sidechain `Balance` smart
+  // contract. If omitted, client implementation MUST set it to the request's
+  // signer `OwnerID`.
```

Comment:
I am not sure if ` If omitted, client implementation MUST set it to the request's signer OwnerID` note is necessary in protocol description. By body definition, request does not suggest you to omit this field.

Reply:
Both protobuf encoding and NeoFS API technically allow it to be omitted or set to zero. Added this note to emphasize the expected default behaviour.